src/Sylius/Bundle/AdminBundle/Resources/views/Customer/Show/_content.html.twig

```diff
@@ -43,4 +43,9 @@
             </div>
         </div>
     </div>
+    {% if customer.orders is not empty %}
```

Comment:
Why? I'd always display the stats. Otherwise some may not be aware of this feature.